server/routerlicious/packages/routerlicious-base/src/riddler/tenantManager.ts

```diff
@@ -57,6 +60,9 @@ export interface ITenantDocument {
 	// Timestamp of when this tenant will be hard deleted.
 	// Only applicable if the tenant is disabled.
 	scheduledDeletionTime?: string;
+
+	// Optional private keys, used for keyless tenants
```

Comment:
Even (key, secondaryKey)  can be optional. However, both (key, secondaryKey) and ITenantPrivateKeys cannot be optional. Only one can be at a given time.

Reply:
As per our discussion, will address in a future change to ensure that internal services do not fetch tenant keys from RIddler.